README-sensors-end-to-end-demo.md

```diff
@@ -124,11 +124,23 @@ The steps in this section only need to be done once to setup and install the nec
 
 The remaining steps in this section are optional for using Kafka Connect:
 
-1. Using an editor, open `docker-compose.yml`. Change `kafka.example.com` to the address of your Kafka broker or cluster. Save the file.
+1. Using an editor, open `docker-compose.yml`.
 
-1. Using an editor, open `connect-mqtt-sensors.json`. Change `kafka.example.com` to the address of your Kafka broker or cluster. Save the file.
+   - Change `kafka.example.com` to the address of your Kafka broker or cluster.
+   - If your Kafka requires the client to use a keystore or truststore, remove the comments from the environment variables prefix with `CONNECT_SECURITY_*`, `CONNECT_SSL_*`, `CONNECT_PRODUCER_SECURITY_*`, and `CONNECT_PRODUCER_SSL_*`, and set the appropriate values.
+   - Save the file.
 
-1. Using an editor, open `connect-mqtt-state.json`. Change `kafka.example.com` to the address of your Kafka broker or cluster. Save the file.
+1. Using an editor, open `connect-mqtt-sensors.json`.
+
+   - Change `kafka.example.com` to the address of your Kafka broker or cluster.
+   - If your Kafka requires the client to use a keystore or truststore, remove the comments from the environment variables prefix with `confluent.topic.security.*`, `confluent.topic.ssl.*`, and set the appropriate values.
```

Comment:
Those aren't actually commented because it is JSON, right?
Suggest defaulting to `"confluent.topic.security.protocol": "PLAINTEXT"` then asking them to change to `"confluent.topic.security.protocol": "SSL"` here in the instructions if needed.